packages/core/docs/welcome.stories.ts

```diff
@@ -23,7 +23,7 @@ export const core = () => {
       </h1>
       <h2 cds-text="section">Web Component Implementation of the Clarity Design System</h2>
 
-      <div cds-layout="vertical gap:lg container:xs" cds-text="body left">
+      <div cds-layout="vertical gap:lg container:xs" cds-text="left">
```

Comment:
This extra body was causing some funky layout issues since the `p` elements below already have them.